docs/yamlpages/solvents.rst

```diff
@@ -0,0 +1,287 @@
+
+* ``NoCutoff``: **Default**. Specifies a non-periodic system with no cutoff. 
+  This is the de facto choice for vacuum and implicit solvent.
+* ``CutoffNonPeriodic``: Specify a non-periodic system which uses a cutoff scheme to reduce the computational overhead of computing long range interactions. 
+  This scheme is helpful for vacuum and implicit solvent systems with large number of molecules where computing the pairwise interaction at long range between all atoms is inefficient.
+* ``CutoffPeriodic``: Specifies a periodic system with a cutoff scheme and Reaction Field electrostatics for long range interactions.
+  There is some cutoff based error associated with using this option and is not currently recommended.
+* ``Ewald``: Specifies a periodic system with a cutoff scheme and Ewald decomposed electrostatics. This is typically not used over PME
+* ``PME``: Specifies a periodic system with a cutoff scheme and Particle Mesh Ewald (PME) decomposed electrostatics. 
+  Currently support by YANK, however, there is a small error introduced by YANK due to the inability efficiently treat long range alchemical PME electrostatics during simulation.
+  We partially correct for this error by computing it at run-time at the cost of a bit of computational overhead.
+  **This is YANK's currently prefered mode for explicit solvent.**
+
+
+.. _yaml_solvents_nonbonded_cutoff:
+
+nonbonded_cutoff
+----------------
+.. code-block:: yaml
+
+   solvents:
+     {UserDefinedSolvent}:
+       nonbonded_cutoff: 1 * nanometer
+
+Specify the cutoff radius for the ``nonbonded_method``\s which rely on it. 
+What happens beyond this cutoff depends both on the ``nonbonded_method`` and the ``switch_distance``.
+
+Nonbonded Methods: ``CutoffPeriodic``, ``Ewald``, ``PME``
+
+Valid Options (1 * nanometer): <Quantity Length> [1]_
+
+
+.. _yaml_solvents_switch_distance:
+
+switch_distance
+---------------
+.. code-block:: yaml
+
+   solents:
+      {UserDefinedSolvent}:
+         switch_distance: 0.9 * nanometer
+
+The distance at which the potential energy switching function is turned on for Lennard-Jones interactions. 
+If the ``switch_distance`` is 0 (or not specified), no switching function will be used. 
+Values greater than ``nonbonded_cutoff`` or less than 0 raise errors.
+
+Nonbonded Methods: ``CutoffPeriodic``, ``Ewald``, ``PME``
+
+Valid Options (0 * nanometer) <Quantity Length> [1]_
+
+
+.. _yaml_solvents_constraints:
+
+constraints
+-----------
+.. code-block:: yaml
+
+   solvents:
+     {UserDefinedSolvent}:
+       constraints: Hbonds
+
+.. todo:: Figure out which ``constraints`` option overwrites the other.
+
+Specify constraints over the entire system. This option is redundant with the general options for :ref:`constraints <yaml_options_constraints>` and should almost never be set here unless you want to specify two solvents with different constraints.
```

Comment:
Good point! We could think of adding an exception for this option in the schema validator.